### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by Debas-31**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+const popupProject = document.querySelector('.popup-project');
+const overlayDiv = document.querySelector('.overlay');
+
+const projects = [
+  { id: 0,
+    title: 'Multi-Post stories',
+    picture: 'image/Img-placeholder-project.png',
+    modalPicture: 'image/Img-placeholder-project-moblie.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe..",
+    technologies: ['html', 'Ruby', 'css'],
```
Comment (TouringIdk) [inline app.js:14]: in your projects arrays you have objets with the propiety technologies, for the languages of your projects, that is good

File: app.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    liveLink: 'https://google.com',
+    codeLink: 'https://github.com',
+  
+  },
+  { id: 1,
+    title: 'Usb flash drive',
+    picture: './image/Img-placeholder-project.png',
```
Comment (eroiyo) [inline app.js:20]: also, the placeholder photo is non-existant in your folder, it shoul be there to be able to appear in the page, wish you the best!

**Review by TouringIdk — CHANGES_REQUESTED** [review]

File: app.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    modalPicture: 'image/Img-placeholder-project.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe..",
+    technologis: ['node.js', 'bootstrap', 'sass' ],
+    liveLink: 'https://google.com',
+    codeLink: 'https://github.com',
+  },
+  { id: 3,
+    title: 'Uber Navigation',
+    picture: 'image/Img-placeholder-project.png',
+    modalPicture: 'image/Img-placeholder-project.png',
```
Comment (eroiyo) [inline app.js:39]: also, the placeholder photo is nonexistent in your folder, it should be there to be able to appear in the page, wish you the best!

File: app.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    technologis: ['node.js', 'bootstrap', 'sass' ],
+    liveLink: 'https://google.com',
+    codeLink: 'https://github.com',
+  },
+  { id: 2,
+    title: 'Facebook 360',
+    picture: 'image/Img-placeholder-project.png',
+    modalPicture: 'image/Img-placeholder-project.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe..",
+    technologis: ['node.js', 'bootstrap', 'sass' ],
```
Comment (TouringIdk) [inline app.js]: but here you are doing a typo error, and it's making the variable to looks like is undefined, kindly fix it

**Review by TouringIdk — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (TouringIdk) [inline app.js]: also, the placeholder photo is nonexistent in your folder, it should be there to be able to appear in the page, wish you the best!

**Pull request closed without merging**